test/web3/web3.js

```diff
@@ -0,0 +1,464 @@
+
+json = methods
+var jsonvalues = Object.values(json);
+web3.currentProvider.enable().then(() => {
+
+  var ethblockNumber = document.getElementById("ethblockNumber");
```

Comment:
Something you could do to reduce code in this file:

Instead of defining each variable and getting it by name, you could iterate over the `json` methods. Something like:

```
Object.keys(json).forEach(methodGroupKey => {
    const methodGroup = json[methodGroupKey]
    Object.keys(methodGroup).forEach(methodKey => {
        const methodButton = document.getElementById(methodKey)
        methodButton.addEventListener('click', function (event) {

          ethereum.sendAsync({
            method: methodKey,
            params: methodGroup[methodKey][1],
          }, function (err, result) {
            if (err) {
              console.log(err);
            } else {
              document.getElementById('results').innerHTML = JSON.stringify(result);
            }
          });
        });
    })
})
  
```

This way you can greatly reduce the code you need in this file.